tests/tests/test_github_webhooks.py

```diff
@@ -0,0 +1,605 @@
+        "debug:Trying to @mention the user in the newly created PR",
+        "debug:userName: tranchitella",
+        'github.CreateComment: org=mendersoftware,repo=workflows-enterprise,number=0,comment={"body":"@tranchitella I have created a PR for you, ready to merge as soon as tests are passed"}',
+        "info:Pull request event with action: closed",
+        "info:workflows:140 would trigger 0 builds",
+        "debug:Got push event :: repo workflows-enterprise :: ref refs/heads/master",
+        "git.Run: /usr/bin/git init .",
+        "git.Run: /usr/bin/git remote add github git@github.com:/mendersoftware/workflows-enterprise.git",
+        "git.Run: /usr/bin/git remote add gitlab git@gitlab.com:Northern.tech/Mender/workflows-enterprise",
+        "git.Run: /usr/bin/git fetch github",
+        "git.Run: /usr/bin/git checkout -b master github/master",
+        "git.Run: /usr/bin/git push -f gitlab master",
+        "info:Pushed ref to GitLab: workflows-enterprise:refs/heads/master",
+    ]
+
+
+def test_issue_comment(integration_test_runner_url):
+    res = requests.post(
+        integration_test_runner_url + "/",
+        data=load_payload("issue_comment.json"),
+        headers={
+            "Content-Type": "application/json",
+            "X-Github-Event": "issue_comment",
+            "X-Github-Delivery": "delivery",
+        },
+    )
+    assert res.status_code == 200
+    #
+    res = requests.get(integration_test_runner_url + "/logs",)
+    assert res.status_code == 200
+    assert res.json() == [
+        "git.Run: /usr/bin/git init .",
+        "git.Run: /usr/bin/git remote add github git@github.com:/mendersoftware/workflows.git",
+        "git.Run: /usr/bin/git remote add gitlab git@gitlab.com:Northern.tech/Mender/workflows",
+        "git.Run: /usr/bin/git fetch github pull/140/head:pr_140",
+        "git.Run: /usr/bin/git push -f --set-upstream gitlab pr_140",
+        "info:Created branch: workflows:pr_140",
+        "info:Pipeline is expected to start automatically",
```

Comment:
(related to my comment in `issue_comment.json`)

I think this does not happen in production, does it? The payload that we have captured has `action: created` but I _think_ is not the case in real payloads from GitHub. I am not sure about this, I haven't checked it now...

Reply:
Same as before, these are copy-paste from GitHub (real webhooks).